var/spack/repos/builtin/packages/py-jupyterhub/package.py

```diff
@@ -11,24 +11,31 @@ class PyJupyterhub(PythonPackage):
 
     pypi = "jupyterhub/jupyterhub-1.0.0.tar.gz"
 
+    version('1.4.1',    sha256='ee1b0718a4db8e0b339796e3e50b704ca6822ab22a7435289dbb5932f65b5199')
     version('1.0.0',    sha256='33541a515a041b9a518ca057c1c4ab4215a7450fdddc206401713ee8137fa67f')
     version('0.9.4',    sha256='7848bbb299536641a59eb1977ec3c7c95d931bace4a2803d7e9b28b9256714da')
 
-    depends_on('python@3.5:', type=('build', 'run'))
-    depends_on('py-setuptools', type=('build', 'run'))
-    depends_on('py-python-dateutil', type=('build', 'run'))
-    depends_on('py-jinja2', type=('build', 'run'))
-    depends_on('py-sqlalchemy@1.1:', type=('build', 'run'))
-    depends_on('py-tornado@5.0:', type=('build', 'run'))
-    depends_on('py-traitlets@4.3.2:', type=('build', 'run'))
     depends_on('py-alembic', type=('build', 'run'))
```

Comment:
I was commenting on the `py-alembic` dependency:
```suggestion
    depends_on('py-alembic@1.4:', type=('build', 'run'))
```